### ISSUE TITLE
Ambassador manipulates/changes  json data types when forwarding to pods

### ISSUE THREAD

**Issue by deepakkhedkar1994** [datawire/ambassador#3161]
**Describe the bug**
when ambassador api gateway forwards json body to pods , it manipulates/ changes json data types . 

**To Reproduce**
when following request is made to eks with ambassador as gateway 
curl --request POST 'http://localhost:9000/predict' 
--header 'Content-Type: application/json' 
--data-raw '{"data":{"ndarray":[{******, **"bbch": 10**, ****}]}}'

when this body reaches to pods it is converted to following 
'{"data":{"ndarray":[{******, **"bbch": 10.0**, ****}]}}'

basically int is converted to float in json body.

**Expected behavior**
data type manipulation should not happen.  

**Versions (please complete the following information):**
 - Ambassador: [1.10.0]
 - Kubernetes environment [EKS 1.17]

**additional info :**
NOTE : i am very new to ambassador and seldon . please ask for more details if needed and please provide relevant commands to produce that details if necessary.

**Comment by mlushpenko** [comment]
not sure if it's related, but in my case, POST requests body data is dropped completely. Post requests were working fine before I've added a payload. And if I make requests via port-forwarding to a service, it works as expected. Also, I can see request body in dev tools, but my backend reports empty form data when request comes via ambassador (installed via helm chart version 6.6.2)
